packages/@glimmer/test-helpers/lib/environment/environment.ts

```diff
@@ -38,6 +39,7 @@ export default abstract class TestEnvironment<TemplateMeta> extends Environment
 
   iterableFor(ref: Reference<Opaque>, keyPath: string): OpaqueIterable {
     let keyFor: KeyFor<Opaque>;
+    check(keyPath, CheckString);
```

Comment:
I'm not sure how helpful this `check` is, seeing it's in test-helpers and it comes right after `keyPath` has its type declared as `string`.

Reply:
I think you can safely remove this, particularly given we have the truthiness check on line 44.